clap_derive/src/lib.rs

```diff
@@ -18,9 +18,14 @@
 #![recursion_limit = "256"]
 
 extern crate proc_macro;
+extern crate syn;
+#[macro_use]
+extern crate quote;
+extern crate heck;
+extern crate proc_macro2;
+extern crate proc_macro_error;
```

Comment:
You don't need `extern` and `macro_use`

Reply:
Yeah, I'll tidy up the imports once we're happy with implementation